src/index.js

```diff
@@ -71,16 +71,28 @@ class App extends React.Component {
     }
   }
 
-  handleWindowLoad() {
-    if(typeof window.web3 !== 'undefined') {
+  async handleWindowLoad() {
+    if(typeof window.ethereum !== 'undefined') {
+      try {
+        window.web3 = new Web3(ethereum);
+        await window.ethereum.enable();
+        this.initialize();
+      } catch (error) {
```

Comment:
I'm sure there will be multiple error types, It should be part of the sanitizeError in util

Reply:
Will wrap the error in sanitizeError.

Reply:
Done. The call to enable will only produce one error as the workflow either succeeds or generates the rejected error in all other flows. 
Also the santizeError method is mainly to handle issues with transactions. I have printed the error in case  of the unknown error scenario as its helpful to know something about the error.